stable/fuse-csi-driver/templates/nodeplugin.yaml

```diff
@@ -0,0 +1,110 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: {{ include "fuse-csi.name.nodeplugin" . }}
+  labels:
+    app: {{ include "fuse-csi.name" . }}
+    component: nodeplugin
+    chart: {{ include "fuse-csi.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ include "fuse-csi.name" . }}
+      component: nodeplugin
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "fuse-csi.name" . }}
+        component: nodeplugin
+        chart: {{ include "fuse-csi.chart" . }}
+        release: {{ .Release.Name }}
+        heritage: {{ .Release.Service }}
+    spec:
+      containers:
+        - name: nodeplugin
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
+            allowPrivilegeEscalation: true
+          image: {{ .Values.csi.v3ioFuse.image.repository }}:{{ .Values.csi.v3ioFuse.image.tag }}
+          imagePullPolicy: {{ .Values.csi.v3ioFuse.image.pullPolicy | quote }}
+          command: ["/bin/v3io-fuse-csi"]
+          args:
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--nodeid=$(NODE_ID)"
+            - "--name=$(DRIVER_NAME)"
+          env:
+            - name: DRIVER_NAME
+              value: {{ .Values.csi.plugin.name }}
+            - name: NODE_ID
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: CSI_ENDPOINT
+              value: "unix:///csi/csi.sock"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+            - name: plugins-dir
+              mountPath: /var/lib/kubelet/plugins
+              mountPropagation: Bidirectional
+            - name: pod-mounts
+              mountPath: /var/lib/kubelet/pods
+              mountPropagation: Bidirectional
+              volumeMounts:
+            - name: v31o-config
+              mountPath: /etc/v3io/fuse 
+              mountPropagation: Bidirectional
+            - name: v3io-log
+              mountPath: /fuse-csi-log
```

Comment:
should be /fuse-log to comply with the multilog path

Reply:
yes, i am wating to the tag from zeek to be made, and then i will push the supported commit in helm-charts